.github/workflows/image-push.yaml

```diff
@@ -0,0 +1,57 @@
+name: Push container image
+on: 
+  push:
+    branches:
+      - main
+    tags:
+      - 'v*.*.*'
+env:
+  image-push-owner: 'alonakaplan'
+jobs:
+  push-amd64:
+    name: Image push/amd64
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to Container Registry
+        if: github.repository_owner == 'alonakaplan'
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Push container image
+        if: github.repository_owner == 'alonakaplan'
```

Comment:
same

Reply:
it is not case sensitive
usually git url part are refereed as lower case